cmd/spire-server/cli/bundle/bundle_test.go

```diff
@@ -109,8 +109,8 @@ func (s *BundleSuite) AfterTest(suiteName, testName string) {
 func (s *BundleSuite) TestShowHelp() {
 	s.showCmd.Help()
 	s.Require().Equal(`Usage of bundle show:
-  -serverAddr string
-    	Address of the SPIRE server (default "localhost:8081")
+  -registrationUDSPath string
+    	Registration API UDS Path (default "./spire_api")
```

Comment:
`./spire_api` conflicts with the default socket path used by the agent for the workload API. Since these are often run on the same box, out of the same folder for development purposes, we might consider a change.